test/test.env

```diff
@@ -5,4 +5,4 @@ COOKIE_MAX_AGE=5400000
 SESSION_ENCRYPTION_KEY=naskjwefvwei72rjkwfmjwfi72rfkjwefmjwefiuwefjkbwfiu24fmjbwfk
 DISABLE_INTERNAL_HTTPS=true
 CSRF_USER_SECRET=123456789012345678
-CARDID_HOST=http://cardid.pymnt.localdomain
+CARDID_HOST=http://cardid.pymnt.localdomain:65530
```

Comment:
Is this ok?

Reply:
Yep, the forward proxy in a test scenario requires a port be available.